### DISCUSSION TITLE
'defaultValue' attribute is not included in migration file

### DISCUSSION THREAD

**Discussion by mandarvl** [mmRoshani/sequelize-typescript-migration#13] [Q&A]
Hi,
I'm using it with a MySQL database and the `defaultValue` field is not used in the `up` command of the generated migration file. I think the bug is in the `reverseModels` function and/or `propertyToStr` function.
How to make it work please?

**Comment by mmRoshani** [comment]
Hey, please provide source code for me so i can test it on my own.

**Reply by mandarvl** [reply]
I'll prepare it then i'll send you a github repository link

**Reply by mandarvl** [reply]
Hi, sorry for the late reply. Here's the repository link [Sequelize Typescript Example](https://github.com/mandarvl/sequelize-typescript-example). Note that you should remove the quote in `"SequelizeMeta"` in the `getLastMigrationState` file.

**Reply by mandarvl** [reply]
I've created a forked that solve my problem with MySQL.
https://github.com/mandarvl/sequelize-typescript-migration
